include/admin_navigation.js

```diff
@@ -442,11 +533,12 @@ module.exports = function AdminNavigationModule(pb) {
             return navigation;
         }
 
-        AdminNavigation.additions = removeNode(id, AdminNavigation.additions);
+        AdminNavigation.additions[site] = removeNode(id, AdminNavigation.additions[site]);
 
-        util.forEach(AdminNavigation.childrenAdditions, function(node, parentId) {
-            AdminNavigation.childrenAdditions[parentId] = removeNode(id, node);
-        });
+        var childAdditionsMap = AdminNavigation.childrenAdditions[site];
+        for (var parentId in  childAdditionsMap) {
+            childAdditionsMap[parentId] = removeNode(id, childAdditionsMap[parentId]);
+        }
```

Comment:
Code climate likes to complain about this kind of "for" loop:
```
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.
```

To solve this we created a utility function:
```
util.forEach(childAdditionsMap, function(value, key) {
  childAdditionsMap[key] = removeNode(id, value);
});
```